build-gw-images.sh

```diff
@@ -145,6 +153,8 @@ retry imagesets/imageset.sh aws update generic-worker-win2016-amd
 retry tc-admin apply
 retry imagesets/imageset.sh aws update generic-worker-win2022
 retry tc-admin apply
+retry imagesets/imageset.sh azure update generic-worker-win2022
+retry tc-admin apply
```

Comment:
Note, that this code looks good to me now, but this probably shouldn't be ran just yet since the azure provider isn't set up on community tc yet.

Reply:
i'll leave it in there for now, and just comment it out locally until we have the provider set up